python/cudf/cudf/utils/applyutils.py

```diff
@@ -23,6 +25,11 @@
 kwargs: dict
     name-value of extra arguments.  These values are passed
     directly into the function.
+pessimistic_nulls : bool
+    Whether or not apply_rows should output nulls when an input is
+    null. If false, all outputs will be non-null, but will be the
```

Comment:
I think we could enhance this a bit by clarifying that this controls the behavior when _any_ of the input values for a row are null.

Reply:
Also, false -> False

Reply:
I've gone over it a few times, and here's the best I've come up with without being overly verbose.

```
Whether or not apply_rows output should be null when any corrosponding
    input is null. If False, all outputs will be non-null, but will be the
    result of applying func against the underlying column data, which
    may be garbage.
```